.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "jellyfinDir"     : "${workspaceFolder}/../jellyfin/Jellyfin.Server",
+    "jellyfinWebDir"  : "${workspaceFolder}/../jellyfin-web",
+    "jellyfinDataDir" : "${env:LOCALAPPDATA}/jellyfin",
```

Comment:
This corresponds to `XDG_DATA_HOME/jellyfin` under Linux ( or `$HOME/.local/share/jellyfin`).

Reply:
Sure, someone else can improve this - The good news is that you can give platform specific configs. 

This still constitutes a massive improvement over the hardcoded docs. 
I don't intend to add to it, so you can fix it if you want, but I suggest you merge it.

Reply:
> Sure, someone else can improve this - The good news is that you can give platform specific configs.


Interesting. Is there any documentation about platform specific configs?

> I don't intend to add to it, so you can fix it if you want, but I suggest you merge it.


I'm not sure I could fix it in this PR, but I might be able to create a follow-up PR should someone from the team merge this one.